ci-operator/config/openstack-k8s-operators/neutron-operator/openstack-k8s-operators-neutron-operator-master.yaml

```diff
@@ -9,6 +9,26 @@ build_root:
     name: release
     namespace: openshift
     tag: rhel-8-release-golang-1.18-openshift-4.12
```

Comment:
Missed to bump golang 1.19 image.

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/36052/rehearse-36052-pull-ci-openstack-k8s-operators-neutron-operator-master-neutron-operator-build-deploy/1621872625884073984#1:build-log.txt%3A448

go: go.mod file indicates go 1.19, but maximum version supported by tidy is 1.18

Reply:
done